docs-v2/reference/integration-configuration.mdx

```diff
@@ -20,6 +20,7 @@ integrations:
         endpoint: # Generated endpoint to access the synced model.
           method: GET
           path: /tasks
+          entity: Tasks
```

Comment:
Not a huge fan of the name `entity`. I don't find it super descriptive. what about `group` or `tag`?

Reply:
I don't either, not my call though. @khaliqgant

Reply:
![CleanShot 2024-11-13 at 17 01 33@2x](https://github.com/user-attachments/assets/4e879743-3fd2-4dc3-b399-bba4ae6f0620)

Tags seem to be the way OpenAPI calls this. It's also a good idea to make it an array, though we don't need this now.

https://swagger.io/docs/specification/v3_0/grouping-operations-with-tags/

Reply:
indeed tags plural could be "useful" but realistically I have never seen an API needing that and it doesn't convey the same meaning. 
By order of pref: Group, Entity, Tag

Reply:
Renamed to group